src/panels/lovelace/views/hui-view.ts

```diff
@@ -96,11 +96,7 @@ export class HUIView extends LitElement {
     element.addEventListener(
       "ll-badge-rebuild",
```

Comment:
We could just use the same event we use for cards here, instead of a separate one for badges?

Reply:
do we want to rebuild cards if we don't have to?